krabs/krabs/etw.hpp

```diff
@@ -252,37 +250,35 @@ namespace krabs { namespace details {
     }
 
     template <typename T>
-    void trace_manager<T>::unregister_trace()
+    void trace_manager<T>::stop_trace()
     {
-        if (trace_.registrationHandle_ != INVALID_PROCESSTRACE_HANDLE)
-        {
-            trace_info info = fill_trace_info();
-            ULONG status = ControlTrace(NULL,
-                trace_.name_.c_str(),
-                &info.properties,
-                EVENT_TRACE_CONTROL_STOP);
-
-            trace_.registrationHandle_ = INVALID_PROCESSTRACE_HANDLE;
+        trace_info info = fill_trace_info();
+        ULONG status = ControlTrace(
+            NULL,
+            trace_.name_.c_str(),
+            &info.properties,
+            EVENT_TRACE_CONTROL_STOP);
+
+        if (status != ERROR_WMI_INSTANCE_NOT_FOUND) {
             error_check_common_conditions(status);
         }
     }
 
     template <typename T>
     EVENT_TRACE_PROPERTIES trace_manager<T>::query_trace()
     {
-        if (trace_.registrationHandle_ != INVALID_PROCESSTRACE_HANDLE)
-        {
-            trace_info info = fill_trace_info();
-            ULONG status = ControlTrace(
-                trace_.registrationHandle_,
-                trace_.name_.c_str(),
-                &info.properties,
-                EVENT_TRACE_CONTROL_QUERY);
+		trace_info info = fill_trace_info();
```

Comment:
nit - please convert tabs to spaces here and elsewhere

Reply:
Done. And fixed my editor settings.